cherrytrack/models.py

```diff
@@ -92,7 +92,7 @@ class SourcePlateWell(db.Model):  # type: ignore
 class AutomationSystemRun(db.Model):  # type: ignore
     __tablename__ = "automation_system_runs"
     __table_args__ = (
-        Index("automation_system_run_id", "system_run_id", "automation_system_id", unique=True),
```

Comment:
This needs a corresponding update in a migration or update the existing migrations and recreate the db since it isn't live yet.